src/core/WakaTimeCore.ts

```diff
@@ -348,6 +349,14 @@ class WakaTimeCore {
     return payload;
   }
 
+  getOperatingSystem(): Promise<string> {
+    return new Promise((resolve) => {
+      chrome.runtime.getPlatformInfo(function(info) {
```

Comment:
I am not sure this works for FF
We might want to use the browser pollyfill for this

Reply:
Testing `chrome`  in FF, it is not defined 
<img width="188" alt="image" src="https://user-images.githubusercontent.com/1050904/234886204-25cf6aa0-0b8d-4dd6-8537-b28387b59b00.png">